Projects/My_website/myWebsite/blog/views.py

```diff
@@ -18,3 +19,11 @@ class crudBlogApi(viewsets.ModelViewSet):
     serializer_class = blogSerializer
```

Comment:
@codal-uchoudhary You have used `ModelViewSet`, so can we use if class for updating the Post instance by providing post_id? if yes, then how?

Reply:
@abhi-codal i will try to find answer of this query . but for Post model i think , i have to implement the API using APIView class , because APIView class give me better control on methods such as Patch, POST, GET ect.